crates/rpc/src/v05.rs

```diff
@@ -12,26 +12,29 @@ pub fn register_routes() -> RpcRouterBuilder {
     RpcRouter::builder("v0.5")
         .register("starknet_blockHashAndNumber"              , v02_method::block_hash_and_number)
         .register("starknet_blockNumber"                     , v02_method::block_number)
+        .register("starknet_call"                            , v02_method::call)
         .register("starknet_chainId"                         , v02_method::chain_id)
         .register("starknet_getBlockTransactionCount"        , v02_method::get_block_transaction_count)
-        .register("starknet_getBlockWithTxHashes"            , method::get_block_with_tx_hashes)
-        .register("starknet_getBlockWithTxs"                 , method::get_block_with_txs)
         .register("starknet_getClass"                        , v02_method::get_class)
         .register("starknet_getClassAt"                      , v02_method::get_class_at)
         .register("starknet_getClassHashAt"                  , v02_method::get_class_hash_at)
         .register("starknet_getNonce"                        , v02_method::get_nonce)
         .register("starknet_getStorageAt"                    , v02_method::get_storage_at)
         
+        .register("starknet_estimateFee"                     , v03_method::estimate_fee)
         .register("starknet_getEvents"                       , v03_method::get_events)
         .register("starknet_getStateUpdate"                  , v03_method::get_state_update)
 
         .register("starknet_addDeclareTransaction"           , v04_method::add_declare_transaction)
         .register("starknet_addDeployAccountTransaction"     , v04_method::add_deploy_account_transaction)
         .register("starknet_addInvokeTransaction"            , v04_method::add_invoke_transaction)
+        .register("starknet_estimateMessageFee"              , v04_method::estimate_message_fee)
         .register("starknet_getTransactionByBlockIdAndIndex" , v04_method::get_transaction_by_block_id_and_index)
         .register("starknet_getTransactionByHash"            , v04_method::get_transaction_by_hash)
         .register("starknet_syncing"                         , v04_method::syncing)
 
+        .register("starknet_getBlockWithTxHashes"            , method::get_block_with_tx_hashes)
+        .register("starknet_getBlockWithTxs"                 , method::get_block_with_txs)
```

Comment:
I've done exactly this in my other PR too so either of us will have a conflict? ;)

Reply:
Go ahead and merge yours first, mine will be much less to rebase.